xml/gl.xml

```diff
@@ -28849,6 +28850,19 @@ typedef unsigned int GLhandleARB;
             <param group="TextureEnvParameter"><ptype>GLenum</ptype> <name>pname</name></param>
             <param len="COMPSIZE(pname)">const <ptype>GLfixed</ptype> *<name>params</name></param>
         </command>
+        <command>
+            <proto>void <name>glTexEstimateMotionQCOM</name></proto>
+            <param group="Texture"><ptype>GLuint</ptype> <name>ref</name></param>
+            <param group="Texture"><ptype>GLuint</ptype> <name>target</name></param>
+            <param group="Texture"><ptype>GLuint</ptype> <name>output</name></param>
```

Comment:
Are all of these meant to be textures? iirc `group="Texture"` translates to `an OpenGL handle of type Texture`

Reply:
yes, those are all texture handles.

Reply:
Ah, ignore me :P